scripts/vulnerabilities.sh

```diff
@@ -16,31 +16,23 @@
 # This script prints a table of fixable vulnerabilities of medium severity or
 # higher, based on the latest or specified git tag.
 #
-# USAGE: $ scripts/vulnerabilities.sh [IMAGE_TAG]
```

Comment:
The optional image tag was added so you could scan a version that wasn't checked out.
Now you can't do that...

Reply:
The input is the rendered manifest now, so the recommended path would be to pull down the intended manifest from GCS

Reply:
Also as a side note it seems like before this change you did actually need to check out the version to get a valid scan result, since it was using the unrendered manifests from the current checkout (for git-sync et al)